scripts/driver.requirements.txt

```diff
@@ -13,4 +13,6 @@ markupsafe < 2.1
 Jinja2 >= 2.10.1
 # Jinja2 >=2.10, <3.0 needs a separate package for type annotations
 types-Jinja2
-
+# jsonschema > 3.2.0 is not compatible with Python 3.5, force 3.2.0
+jsonschema >= 3.2.0
```

Comment:
Is that a reference to [this ](https://github.com/python-jsonschema/jsonschema/issues/228)bug? 

At any rate this line is doing the oppposite to what the message says, which is installing the highest available version of  jsonschema over 3.2.0

Was it meant to be `jsonschema <= 3.2.0` or `jsonschema > 3.2.0` ?

Reply:
JsonSchema versions greater than 3.2.0 are not compatible with Python 3.5. There were some CI servers where the python version was different ( It think lower and this caused some failures), to get around CI, I originally forced the version to 3.2.0. The CI server was later fixed, so now marking >=3.2.0 ensures minimum rev 3.2.0 as compatible with the stock Python on the device ( mbedTLS recommends 3.5). For higher versions of Python, when we upgrade the packages should automatically be compatible.

Will knock off the comment, if it seems misleading.

Reply:
You can remove it or reword it. I have no strong prefference as long as the comment describes the behavior we are seeing in the code, or conveys that extra information.

>Some Python 3.5 enviroments may have trouble installing jsonschema < 3.2.0. Fix it to jsonschema = 3.2.0 if required.

Reply:
I've removed the comment.
FYI, it was originally "jsonschema = 3.2.0", was updated to  "jsonschema >= 3.2.0" after CI server was fixed with  appropriate Python version and based on comments from @gilles-peskine-arm 
https://github.com/Mbed-TLS/mbedtls/pull/5396#discussion_r798593543